uqcsbot/base.py

```diff
@@ -193,8 +193,12 @@ def _handle_command(self, message: dict) -> None:
         command = Command.from_message(message)
         if command is None:
             return
-        for handler in self._command_registry[command.name]:
-            self.executor.submit(self._execute_catching_error, handler, command)
+        handlers = self._command_registry[command.name]
+        if not handlers:
+            self.post_message(command.channel_id, "Unknown command. Use !help to retrieve a list of supported commands")
```

Comment:
Split up the String argument onto the next line so that Jenkins doesn't disapprove. It currently says 
> uqcsbot/base.py:198:101: E501 line too long (120 > 100 characters)